### ISSUE TITLE
Parallel Machines

### ISSUE THREAD

**Issue by notmuchnerdy** [dwave-examples/job-shop-scheduling-cqm#10]
Dear developers,

Thank you for your impressive effort on this scheduling problem. 
I would like to modify the code due to my problem but I wanted to be sure in advance if this model is capable of this problem.

I want to explain it with an example.  
1) I have 16 jobs (each job has 1 task) and 3 parallel machines (all of the jobs can be done by three parallel machines with identical processing times.)
2) There is no specific sequence for the jobs and we think that there is no precedence between jobs.

Basically, all of the jobs (16 jobs=16 tasks) have to be processed on three identical machines without considering any sequential order. Each job has to be started and done in one machine. In your model, the solution can be found if only if the number of tasks is equal to the number of machines. May I somehow bend this rule? 

Sincerely.

**Comment by k8culver** [comment]
Hi @notmuchnerdy, because this is not a bug or feature request, but rather a question of how to solve a problem, these sorts of questions will be answered better and more quickly in [our community](https://support.dwavesys.com/hc/en-us/community/topics).
